pyopia/background.py

```diff
@@ -108,17 +110,26 @@ def correct_im_accurate(imbg, imraw):
         background averaged image
     imraw : float64
         raw image
+    divide_bg : (bool, optional)
+        If True, the correction will be performed by dividing the raw image by the background
+        Default to False
 
     Returns
     -------
     im_corrected : float64
         corrected image, same type as input
     '''
 
-    im_corrected = imraw - imbg
-    im_corrected += (1 / 2 - np.percentile(im_corrected, 50))
-
-    im_corrected += 1 - im_corrected.max()
+    if divide_bg:
+        imbg = np.clip(imbg, a_min=1/255, a_max=None)   # Clipping the zero_value pixels
+        im_corrected = imraw / imbg
+        im_corrected += (1 / 2 - np.percentile(im_corrected, 50))
+        im_corrected -= im_corrected.min()   # Shift the negative values to zero
```

Comment:
This line is doing the same thing as the next one I think, and could be removed.